examples/p2p-webrtc/voice-agent/bot.py

```diff
@@ -36,9 +40,11 @@
 
 
 async def run_bot(webrtc_connection):
-    pipecat_transport = SmallWebRTCTransport(
-        webrtc_connection=webrtc_connection,
-        params=TransportParams(
+    pipecat_transport = PipecatCloudTransport(
+        session_args=SessionArguments(
+            webrtc_connection=webrtc_connection,
+        ),
+        params=PipecatCloudParams(
```

Comment:
I think these examples inside `p2p-webrtc` should continue using `SmallWebRTCTransport`, since the intention here was to be didactic and provide an easy way to understand how to use this transport.

Maybe we could create a new folder inside the examples, like `pcc`, or even better, as @aconchillo  suggested a public repo for `pipecat-cloud`, where we would add both the new transport `PipecatCloudTransport`  and all the examples using it ?